packages/examples/server/src/server.ts

```diff
@@ -40,6 +40,7 @@ server.on('upgrade', (request: http.IncomingMessage, socket: net.Socket, head: B
         wss.handleUpgrade(request, socket, head, webSocket => {
             const socket: rpc.IWebSocket = {
                 send: content => webSocket.send(content, error => {
+                    console.log(content);
```

Comment:
is it there on purpose?

Reply:
Yes, I put this in to see if content is sent and I thought it may be helpful, because it is an example, but we can remove it again,